src/controller_examples/rabbitmq_controller/spec/rabbitmqcluster.rs

```diff
@@ -110,12 +110,14 @@ impl ResourceView for RabbitmqClusterView {
 
     open spec fn transition_rule(new_obj: RabbitmqClusterView, old_obj: RabbitmqClusterView) -> bool {
         new_obj.spec.replicas >= old_obj.spec.replicas
+        && new_obj.spec.persistence == old_obj.spec.persistence
```

Comment:
This might be a nuance, but can you change the validation rule to check persistence.storage and persistence.storage_class_name?